### ISSUE TITLE
bonds price history doesnt work 

### ISSUE THREAD

**Issue by Aks1623** [ymyke/tessa#5]
firstly Thanks for trying to find a solution around the 429 error , it is the bane of my existence for now .. i am glad i found this package .

the function doesn't work for Bonds

i attach the snapshot below of what the price history function throws vs what investpy api says 

<img width="1235" alt="Screenshot 2022-05-23 at 10 05 27" src="https://user-images.githubusercontent.com/38810502/169773340-ca94d374-b4ab-49ea-8298-60d3497355ec.png">

**Comment by ymyke** [comment]
Thanks for pointing this out. It seems that the bond type is buggy in investpy. I reported it here: https://github.com/alvarobartt/investpy/issues/572

But I made the type work for tessa:
```python
from tessa import price_history
df, currency = price_history("Australia 30Y", "bond")
```

This will return the prices, albeit with "UNKNOWN" currency, since the currency column is missing in the dataframe that investpy returns.

Also, I'm not sure which and how many bonds work correctly since investpy currently doesn't support the `country` param in `get_bond_historical_data`.

...

**Love this: "the 429 error , it is the bane of my existence for now"** 🤣🤣 That made my day.
